mavenTest/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>hillel.example</groupId>
+    <artifactId>mavenTest</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>20</maven.compiler.source>
+        <maven.compiler.target>20</maven.compiler.target>
+        <testng.version>7.8.0</testng.version>
+        <maven.surefire.version>3.1.2</maven.surefire.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+
+    </properties>
+
+    <profiles>
+        <profile>
+            <id>dev</id>
+
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>3.1.2</version>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <url>https://dev.google.com</url>
+                                <my.value>Hello</my.value>
+                            </systemPropertyVariables>
+                            <!--                    <suiteXmlFiles>-->
+                            <!--                        <suiteXmlFile>testng.xml</suiteXmlFile>-->
+                            <!--                    </suiteXmlFiles>-->
```

Comment:
Please, do not leave commented parts of code. It makes no sense and it's a bad practice.